script/setup/install-cni

```diff
@@ -21,7 +21,14 @@
 #
 set -eu -o pipefail
 
-CNI_COMMIT=${1:-$(grep containernetworking/plugins "$GOPATH"/src/github.com/containerd/containerd/go.mod | awk '{print $2}')}
+if [ -f "go.mod" ]; then
+  CNI_COMMIT=${1:-$(grep containernetworking/plugins go.mod | awk '{print $2}')}
```

Comment:
It is not correct. The script can be called in any where. 
It doesn't work if you don't use relative path here.

Suggest to use the following script(for example) to get the repo root path.

```bash
REPO_ROOT="$(cd $(dirname "${BASH_SOURCE[0]}")/../.. && pwd -P)"

...

grep containernetworking/plugins ${REPO_ROOT}/go.mod
```

Reply:
Thanks for your suggestion, I feel good.